docs/source/algorithms/CreatePoleFigureTableWorkspace-v1.rst

```diff
@@ -0,0 +1,92 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Create a table with the information required to produce a pole figure, namely, a collection of alphas, betas,
+and intensities.
+
+If a peak parameters table is provided, the intensities and chi squared values are used by the algorithm,
+otherwise the chi squared will be assumed to be ``0`` and the intensities assumed to be ``1``.
+
+The algorithm supports selection of which spectra from the input are included in the final table, by use of both
+a threshold on the peak position and on the chi squared value. If a reflection is specified, the algorithm will
+check that the fit ``X0`` value in the peak parameter table is within ``PeakThreshold`` of the d spacing value
+for the specified peak (Note: as such, specifying a reflection requires a
+:ref:`Crystal Structure concept page <Crystal structure and reflections>` be set on the Input Workspace). If a chi squared threshold is
```

Comment:
If a `CrystalStructure` and `hkl` are provided you also calculate/normalise by the scattering power - i think this should be mentioned as well, with a link to the NyRTeX paper.